api/envoy/extensions/filters/network/zookeeper_proxy/v3/zookeeper_proxy.proto

```diff
@@ -42,19 +42,23 @@ message ZooKeeperProxy {
   // if that is set. If it isn't, ZooKeeper's default is also 1Mb.
   google.protobuf.UInt32Value max_packet_bytes = 3;
 
+  // Whether to emit latency threshold metrics. If not set, defaults to false.
+  // If false, setting `default_latency_threshold` and `latency_threshold_overrides` will not have effect.
+  bool enable_latency_threshold_metrics = 4;
```

Comment:
I think we'd want to leave the previous numeric IDs the same and make this new one be number 6.

Reply:
Since both 5 and 6 are also newly added, I think it should be ok to make `enable_latency_threshold_metrics` as 4?
The main reason is that with current order, the proto doc could be a bit easier to read. We put the toggle (`enable_latency_threshold_metrics`) first, then the detailed settings.

Reply:
Oh sorry I was looking only at the diff in the most recent commit. What you have is fine :)